docs-partials/builder/vsphere/iso/CreateConfig-not-required.mdx

```diff
@@ -9,7 +9,7 @@
   here](https://developer.vmware.com/apis/358/vsphere/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html)
   for a full list of possible values.
 
-- `network_adapters` ([]NIC) - Sets the network adapters, if no adapter is specified, every network related task is not possible.
+- `network_adapters` ([]NIC) - Sets the network adapters, if no adapter is specified, every network related task is not applicable.
```

Comment:
This update would be generated with `packer-sdc generate`.

Reply:
I don't think I know what you mean, there is not `generate` command for packer-sdc